CONTRIBUTING.md

```diff
@@ -0,0 +1,244 @@
+# Apache Ozone Website Contribution Guide
+
+Ozone is an Apache project. The bug tracking system for Ozone and its website is under the [Apache Jira project named HDDS](https://issues.apache.org/jira/projects/HDDS/).
```

Comment:
The Apache Ozone is a top-level Apache project and is licensed under the [Apache 2.0 License](https://www.apache.org/licenses/LICENSE-2.0).